hazelcast/src/main/java/com/hazelcast/multimap/impl/operations/ClearBackupOperation.java

```diff
@@ -19,8 +19,9 @@
 import com.hazelcast.multimap.impl.MultiMapContainer;
 import com.hazelcast.multimap.impl.MultiMapDataSerializerHook;
 import com.hazelcast.spi.BackupOperation;
+import com.hazelcast.spi.impl.MutatingOperation;
 
-public class ClearBackupOperation extends MultiMapOperation implements BackupOperation {
+public class ClearBackupOperation extends MultiMapOperation implements BackupOperation, MutatingOperation {
```

Comment:
Is quorum needed for a `BackupOperation`? If primary applies the write, backups should write too. What is the benefit of allowing writes on primary but rejecting a backup?

Reply:
As discussed on slack.